.github/workflows/node-libs.yml

```
@@ -120,18 +120,13 @@ jobs:
       - name: Get commit message and branch name
         id: git
         env:
-          COMMIT_MESSAGE_ENV: ${{ github.event.head_commit.message }}
+          COMMIT_MESSAGE_ENV: ${{ github.event.pull_request.title || github.event.head_commit.message }}
           EVENT_NAME_ENV: ${{ github.event_name }}
           BRANCH_NAME_ENV: ${{ github.head_ref || github.ref_name }}
         run: |
-          LOG=""
+          LOG=$COMMIT_MESSAGE_ENV
           EOF_MARKER=$GITHUB_SHA # github env var
```

everything is env variables now => lets remove the comment 😄 
```suggestion
          EOF_MARKER=$GITHUB_SHA
```